package.json

```diff
@@ -1,145 +1 @@
esforce","bin":{"oclif":"bin/run"},"bugs":"https://github.com/oclif/oclif/issues","dependencies":{"@oclif/core":"^1.20.4","@oclif/plugin-help":"^5.1.19","@oclif/plugin-not-found":"^2.3.7","@oclif/plugin-warn-if-update-available":"^2.0.14","aws-sdk":"^2.1231.0","concurrently":"^7.5.0","debug":"^4.3.3","find-yarn-workspace-root":"^2.0.0","fs-extra":"^8.1","github-slugger":"^1.5.0","got":"^11","lodash":"^4.17.21","normalize-package-data":"^3.0.3","semver":"^7.3.8","tslib":"^2.3.1","yeoman-environment":"^3.11.1","yeoman-generator":"^5.6.1","yosay":"^2.0.2"},"devDependencies":{"@oclif/plugin-legacy":"^1.2.7","@oclif/test":"^2.2.12","@types/chai":"^4.3.4","@types/execa":"^0.9.0","@types/fs-extra":"^9.0","@types/lodash":"^4.14.186","@types/lodash.template":"^4.5.0","@types/mocha":"^8.2.3","@types/node":"^14.18.33","@types/read-pkg":"^5.1.0","@types/semver":"^7.3.13","@types/shelljs":"^0.8.11","@types/supports-color":"^7.2.1","@types/write-json-file":"^3.2.1","@types/yeoman-generator":"^5.2.10","@types/yosay":"^2.0.1","chai":"^4.3.7","conventional-changelog-cli":"^2.2.2","eslint":"^7.32.0","eslint-config-oclif":"^4.0.0","eslint-config-oclif-typescript":"^1.0.2","execa":"^0.11.0","fancy-test":"^1.4.10","globby":"^11.1.0","mocha":"^9.2.2","npm-run-path":"^4.0.1","nps":"^5.10.0","shelljs":"^0.8.5","shx":"^0.3.4","tmp":"^0.2.1","ts-node":"^10.7.0","typescript":"4.5.5"},"resolutions":{"colors":"1.4.0"},"overrides":{"colors":"1.4.0"},"engines":{"node":">=12.0.0"},"files":[".oclif.manifest.json","/bin","/lib","/templates"],"homepage":"https://github.com/oclif/oclif","keywords":["oclif"],"license":"MIT","main":"lib/index.js","oclif":{"commands":"./lib/commands","plugins":["@oclif/plugin-help","@oclif/plugin-warn-if-update-available","@oclif/plugin-not-found"],"bin":"oclif","dirname":"oclif","topicSeparator":" ","macos":{"identifier":"com.oclif.cli"},"update":{"autoupdate":{"rollout":50,"debounce":60},"node":{"version":"16.13.2"},"s3":{"bucket":"dfc-data-production","indexVersionLimit":20,"folder":"media/salesforce-cli/oclif-testing","acl":" ","host":"https://developer.salesforce.com","xz":true}},"topics":{"pack":{"description":"package an oclif CLI into installable artifacts"},"upload":{"description":"upload installable CLI artifacts to AWS S3"}}},"repository":"oclif/oclif","scripts":{"build":"shx rm -rf lib && tsc","devcli:lint":"eslint . --ext .ts --config .eslintrc","devcli:test":"mocha --forbid-only \"test/unit/*.test.ts\"","devcli":"yarn build --noEmit && yarn run devcli:test && yarn run devcli:lint","lint":"nps lint","postpack":"shx rm .oclif.manifest.json","posttest":"yarn run lint","prepack":"shx rm -rf lib && tsc && bin/dev manifest .","test":"nps test && yarn test:unit && yarn test:integration","test:integration":"mocha --forbid-only \"test/integration/*.test.ts\"","test:unit":"mocha --forbid-only \"test/unit/*.test.ts\"","version":"bin/dev readme && git add README.md"},"publishConfig":{"registry":"https://registry.npmjs.org"},"types":"lib/index.d.ts"}
```

Comment:
???
How'd this lose its formatting? 🤔

Reply:
the tests do that (they also modify the version), and then put it back when they're done.  I probably Ctrl-C'd something.